ui/externs/ui.js

```diff
@@ -357,3 +356,43 @@ shaka.extern.IUISettingsMenu = class {
     this.backSpan;
   }
 };
+
+/**
+ * Interface for SeekBars. SeekBars should inherit from the concrete base
+ * class shaka.ui.Element.
```

Comment:
The default inherits from shaka.ui.RangeElement, which manages some tricky details for various older browsers.  Though you could just as well inherit from shaka.ui.Element, too, if you implement the full interface.  Or you could conceivably inherit from shaka.ui.SeekBar.

I guess the imperative is shaka.ui.Element, so you're not wrong.  But we should probably mention also that you can inherit from shaka.ui.RangeElement or shaka.ui.SeekBar, as well, to save yourself some effort.

Reply:
I updated that comment line accordingly !